t/porting/copyright.t

```diff
@@ -27,6 +27,7 @@ if ( $Config{usecrosscompile} ) {
 }
 
 my ($opt) = @ARGV;
+$opt ||= '';
```

Comment:
this could probably use safely `//=` instead as this is a porting tool?

Reply:
Okay, see 2021c39cbb5f598a82bcd552898cb09c3a0690f3